src/v2/Apps/Fair/Components/FairExhibitors/FairExhibitorsGroup.tsx

```diff
@@ -11,15 +13,34 @@ interface FairExhibitorsGroupProps {
 export const FairExhibitorsGroup: React.FC<FairExhibitorsGroupProps> = ({
   exhibitorsGroup: { exhibitors },
 }) => {
+  const { match } = useRouter()
+  const { focused_exhibitor: focusedExhibitorID } = match.location.query
+  const isMobile = useMatchMedia(themeProps.mediaQueries.xs)
+
+  const focusedExhibitorStyles = {
+    borderColor: "brand",
+    borderStyle: "solid",
+    borderWidth: "1px",
+    padding: "2px",
+  }
+
   return (
     <GridColumns position="relative" gridRowGap={4}>
       {exhibitors?.map(exhibitor => {
         if (!exhibitor?.partner) {
           return null
         }
 
+        const focused = focusedExhibitorID === exhibitor.partner.internalID
+        const exhibitorCardStyle =
+          focused && !isMobile ? focusedExhibitorStyles : {}
```

Comment:
For mobile styles you'd just use responsive props instead of the hook

Reply:
`borderColor: ["brand", undefined]` etc...

Reply:
All this changes deleted. We should not use the border.